### PR TITLE
250 update massa station massa wallet troubleshooting sections

### PR DESCRIPTION
**Pull request by SlnPons**

### CODE REVIEW

File: docs/massaStation/troubleshooting.mdx [hunk 1]
```diff
@@ -175,3 +175,13 @@ and attach the logs to your issue so we can help you.
...
+
+## `Massa Wallet can't be opened`error 
+ADD A SCREENSHOT OF THE ERROR HERE
+So far, this issue happens only on Linux because some dependencies aren't installed correctly.
+
+#### Linux
+1. Open your terminal
+2. Verify witch version of this packet you have `libwebkit2gtk` with `apt list --installed libwebkit2gtk`
+3. The required version is `0.4-37`, so you need to install it with this cmd `sudo apt install libwebkit2gtk-4.0-37`. If the version installed is already the `0.4-37`, please open an issue [here](https://github.com/massalabs/station/issues/new/choose).
+4. Once the library installed, relaunch Massa Station and try to install Massa Wallet again
```
Comment (Thykof) [inline docs/massaStation/troubleshooting.mdx:187]: you can say to restart the wallet plugin by turn it off and on again. This is not related to station, only to the wallet plugin

File: docs/massaStation/troubleshooting.mdx [hunk 2]
```diff
@@ -175,3 +175,13 @@ and attach the logs to your issue so we can help you.
 
 The log file is located in `~/.config/massa-station-wallet/`. You can access it using the Files app or with a terminal using `cd`.
 Once in the directory, search for the file named `station-massa-wallet.log`.
+
+## `Massa Wallet can't be opened`error 
+ADD A SCREENSHOT OF THE ERROR HERE
```
Comment (thomas-senechal) [inline docs/massaStation/troubleshooting.mdx:180]: Who should perform this action ? 
And when ? 
I'm unsure we should push a doc with an action to perform in it ^^

Comment (Thykof) [reply docs/massaStation/troubleshooting.mdx:180]: image

Comment (Thykof) [reply docs/massaStation/troubleshooting.mdx:180]: @SlnPons can you confirm you want this image?

Comment (SlnPons) [reply docs/massaStation/troubleshooting.mdx:180]: yes, it's this one

**Comment by Thykof** [comment]
image

**Review by Thykof — APPROVED** [review]

**Review by qdrn — APPROVED** [review]
